.github/workflows/main.yaml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    tags:
```

Comment:
do we need these on tags? the commit message checker will probably fail here. I would just remove this

Reply:
This is the copy paste from the equivalent file on matrix sdk repo :)

Reply:
Removed